test/test.js

```diff
@@ -72,4 +72,24 @@ describe('Overall test', () => {
             });
     });
 
+
+    it('should not crash even if some fields are null', function (done) {
```

Comment:
add negative test case for array of keys?

Reply:
By test case for array of keys, do you mean this kind of usage:
```js
templater.make
    .template('sample')
    .content({
        date: null,
        hello: {
            trans: [],
            data: {
                name: 'raven'
            }
        }
    })
    .build(function () {
        // ...
    });
```

If so, then I don't think we have to cover this since this is a direct misuse of the library. If ever, I will just wrap it in a should and expect it to throw an error.

Reply:
it's a misuse that is why we need to have a negative test case to document it.